### PR TITLE
Feature/favorites

### PR DESCRIPTION
**Pull request by kosovych**

### CODE REVIEW

File: src/containers/Favorites/DeleteMovie/container.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+import {
+  removeMovie as removeMovieAction,
+  cleanupRemoveMovieErrors as cleanupRemoveMovieErrorsAction
+} from 'Store/favorites/actions'
+import DeleteMovieComponent from './component'
+
+class DeleteMovie extends Component {
+  onDeleteMovie = () => {
+    const { removeMovie, movieId, title } = this.props
+    return Modal.confirm({
```
Comment (VladWithCookies) [inline src/containers/Favorites/DeleteMovie/container.js:17]: do we need ti return it?

File: src/containers/Favorites/DeleteMovie/container.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    const {
+      movieId, title, cleanupRemoveMovieErrors, removeMovieErrors
+    } = this.props
+    const config = {
+      message: 'Error',
+      description: `Can't remove "${title}" the from Favorites`,
+      type: 'error',
+      onClose: () => cleanupRemoveMovieErrors(movieId)
+    }
+    const error = removeMovieErrors.includes(movieId) && openNotification(config)
```
Comment (VladWithCookies) [inline src/containers/Favorites/DeleteMovie/container.js:33]: `openNotification` should be in the operation

File: src/containers/Favorites/DeleteMovie/container.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    return error
+  }
+
+  render() {
+    return (
+      <DeleteMovieComponent onClick={this.onDeleteMovie}>
+        {this.error}
```
Comment (VladWithCookies) [inline src/containers/Favorites/DeleteMovie/container.js:40]: needless line

File: src/store/favorites/selectors.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+export const isBlankSelector = createSelector(
+  movieCatalogEntriesSelector,
+  movieLoadingSelector,
+  (moviesEntries, loading) => !loading && moviesEntries && !moviesEntries.length
+)
+
+export const removeMovieErrorsSelector = createSelector(
+  movieMetaSelector,
+  meta => get(meta, 'removeErrors') || []
```
Comment (VladWithCookies) [inline src/store/favorites/selectors.js:44]: can we do `get(meta, 'removeErrors', [])`?

File: src/store/rootReducer.js [hunk 5]
```diff
@@ -4,12 +4,14 @@ import auth from './auth'
...
 import watchList from './watchList'
+import favorites from './favorites'
 
 const reducers = combineReducers({
   auth,
   data,
   trendingMovies,
-  watchList
+  watchList,
+  favorites
```
Comment (VladWithCookies) [inline src/store/rootReducer.js:14]: watchListMovies
favoriteMovies

File: src/containers/Favorites/DeleteMovie/container.js (hunk 2 above)
Comment (kosovych) [reply src/containers/Favorites/DeleteMovie/container.js:33]: @VladWithCookies 
image

File: src/containers/Favorites/DeleteMovie/container.js (hunk 3 above)
Comment (kosovych) [reply src/containers/Favorites/DeleteMovie/container.js:40]: @VladWithCookies 
image

File: src/containers/Favorites/DeleteMovie/container.js (hunk 2 above)
Comment (VladWithCookies) [reply src/containers/Favorites/DeleteMovie/container.js:33]: Ок

File: src/containers/Favorites/DeleteMovie/container.js (hunk 3 above)
Comment (VladWithCookies) [reply src/containers/Favorites/DeleteMovie/container.js:40]: Ок

File: src/containers/Favorites/DeleteMovie/container.js (hunk 1 above)
Comment (kosovych) [reply src/containers/Favorites/DeleteMovie/container.js:17]: no

File: src/store/favorites/selectors.js (hunk 4 above)
Comment (kosovych) [reply src/store/favorites/selectors.js:44]: @VladWithCookies this code will be removed in the following PR. I'll remake this functionality

**Review by VladWithCookies — APPROVED** [review]

**Pull request merged**